test/extended/router/http2.go

```diff
@@ -142,19 +238,151 @@ var _ = g.Describe("[sig-network-edge][Conformance][Area:Networking][Feature:Rou
 			o.Expect(err).NotTo(o.HaveOccurred())
 
 			g.By("Creating routes to test for http/2 compliance")
-			err = oc.Run("new-app").Args("-f", http2RoutesConfigPath,
-				"-p", "DOMAIN="+shardFQDN,
-				"-p", "TLS_CRT="+pemCrt,
-				"-p", "TLS_KEY="+derKey,
-				"-p", "TYPE="+oc.Namespace()).Execute()
-			o.Expect(err).NotTo(o.HaveOccurred())
+			routeType := oc.Namespace()
+			routes := []routev1.Route{
+				{
+					ObjectMeta: metav1.ObjectMeta{
```

Comment:
In previous commits, you specify `TypeMeta`, but from this commit onwards, you omit it.  Is it necessary in some cases and not in others?

Reply:
I removed all TypeMeta occurrences. I realized a bit late I don't need to set this field.